CHANGES.md

```diff
@@ -76,6 +76,7 @@ To be released.
  -  Added `IProtocol` interface.  [[#1120]]
  -  Added `KademliaProtocol` class which implements `IProtocol`.
     [[#1120], [#1135]]
+ -  Added `IActionContext.IsBlockAction` property. [[#1143]]
```

Comment:
And also I guess we may need to move this section to "Backward-incompatible API changes" because `IActionContext` was open to users.

Reply:
amended https://github.com/planetarium/libplanet/pull/1143/commits/c6d761364eab4aa0297b920d14d3cf534dfc6841